content/en/docs/tasks/tools/install-kubeadm.md

```diff
@@ -234,23 +234,18 @@ kubeadm to tell it what to do.
 
 ## Configure cgroup driver used by kubelet on Master Node
 
-Make sure that the cgroup driver used by kubelet is the same as the one used by Docker. Verify that your Docker cgroup driver matches the kubelet config:
+When using Docker, kubeadm will detect the cgroup driver for the kubelet
```

Comment:
Suggested edit:
"When using Docker, kubeadm will automatically detect the cgroup driver for the kubelet
 and set it in the `/var/lib/kubelet/kubeadm-flags.env` file
during runtime."